src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Errors/ErrorCode.cs

```diff
@@ -96,15 +96,15 @@ internal enum ErrorCode
         ERR_RefReadonlyLocalCause = 1657,
         ERR_DelegateOnNullable = 1728,
         ERR_BadCtorArgCount = 1729,
+        ERR_NamedArgumentSpecificationBeforeFixedArgument = 1738,
+        ERR_BadNamedArgument = 1739,
+        ERR_DuplicateNamedArgument = 1740,
+        ERR_BadNamedArgumentForDelegateInvoke = 1746,
+        ERR_NamedArgumentUsedInPositional = 1744,
         ERR_BadExtensionArgTypes = 1928,
         ERR_BadInstanceArgType = 1929,
         ERR_BadArgTypesForCollectionAdd = 1950,
         ERR_InitializerAddHasParamModifiers = 1954,
         ERR_NonInvocableMemberCalled = 1955,
-        ERR_NamedArgumentSpecificationBeforeFixedArgument = 5002,
-        ERR_BadNamedArgument = 5003,
-        ERR_BadNamedArgumentForDelegateInvoke = 5004,
-        ERR_DuplicateNamedArgument = 5005,
-        ERR_NamedArgumentUsedInPositional = 5006,
```

Comment:
It's curious that these four didn't have the same codes as correspond in roslyn when there's no conflict. There wasn't a conflict in the initial commit that since went away, which would have explained it.